healthtools/views/search_api.py

```diff
@@ -1,12 +1,30 @@
 from flask import Blueprint, request, jsonify
 
 from healthtools.search import run_query
+from healthtools.search.wit_ai import wit_run_query
 
 
 blueprint = Blueprint('search_api', __name__)
 
 
 @blueprint.route('/search', methods=['GET'], strict_slashes=False)
+def search_index(doc_type=None):
```

Comment:
@Awinja-Andela Why are we creating a separate function for what's already below as "index"?

Reply:
@DavidLemayian notice that i have introduced :
`from healthtools.search.wit_ai import wit_run_query`
while still retaining:
`from healthtools.search import run_query`

`@blueprint.route('/wit_search', methods=['GET'], strict_slashes=False)`
that uses:
`result, doc_type = wit_run_query(query, doc_type)`

and
`
@blueprint.route('/search', methods=['GET'], strict_slashes=False)
@blueprint.route('/search/<doc_type>', methods=['GET'], strict_slashes=False)
`
that still uses:
`result, doc_type = run_query(query, doc_type)`